doc/kubernetes/modules/ROOT/partials/aurora/aurora-create-peering-connections.adoc

```diff
@@ -0,0 +1,214 @@
+. Retrieve the Aurora VPC
++
+.Command:
+[source,bash]
+----
+aws ec2 describe-vpcs \
+  --filters "Name=tag:AuroraCluster,Values=keycloak-aurora" \
+  --query 'Vpcs[*].VpcId' \
+  --region eu-west-1 \
+  --output text
+----
++
+.Output:
+[source,json]
+----
+vpc-0b40bd7c59dbe4277
```

Comment:
This output is in red colour which seems little bit weird to me. Not sure why it is red, though.

Reply:
Disregard this comment. In the compiled version it looks good!